packages/host/tests/acceptance/operator-mode-test.ts

```diff
@@ -1388,6 +1391,8 @@ module('Acceptance | operator mode tests', function (hooks) {
       },
     );
 
+    await percySnapshot(assert);
```

Comment:
I think only one snapshot is needed here? Or maybe this is austerity thinking because we’re always using up our allocation 🙃

Reply:
actually- i think we need more than 1. the issue is is that I cannot verify the saving indicator via qunit because of the test waiters (like I mentioned in the description of this PR). and there are specific scenarios that I'm using percy for in order to detect that the saving indicator appears based on the different paths we take via saving (saving card source vs saving card json)